comp/networkpath/npcollector/npcollectorimpl/npcollector.go

```diff
@@ -76,6 +76,9 @@ func newNpCollectorImpl(epForwarder eventplatform.Forwarder, collectorConfigs *c
 		collectorConfigs.pathtestInterval,
 		collectorConfigs.flushInterval)
 
+	getStatsdClient := func() ddgostatsd.ClientInterface {
+		return statsd.Client
```

Comment:
I haven't worked much with `fx` -- should we be injecting the statsd client? In general DI frameworks are supposed to solve these configuration ordering issues.

Reply:
Hi @leeavital , I think you are right, statsd client should be injected as component instead of using a global (currently).

I will reach processes team to see if that's something we can do.